templates/bot/js/notification-function-base/README.md

```diff
@@ -65,6 +65,16 @@ You can find the Teams app manifest in `templates/appPackage/manifest.template.j
 
 The file contains template arguments with `{...}` statements which will be replaced at build time. You may add any extra properties or permissions you require to this file. See the [schema reference](https://docs.microsoft.com/microsoftteams/platform/resources/schema/manifest-schema) for more information.
 
+## Go production
+
```

Comment:
"Go production" seems like a very general term, can we just be more specific? Like "Add authentication for your bot API"?

Reply:
Thanks, I have changed it to `Add authentication for your notification API`